src/libstd/alloc.rs

```diff
@@ -133,24 +133,41 @@ pub use alloc_crate::alloc::*;
 #[derive(Debug, Default, Copy, Clone)]
 pub struct System;
 
-// The AllocRef impl just forwards to the GlobalAlloc impl, which is in `std::sys::*::alloc`.
+// The AllocRef impl currently just forwards to the GlobalAlloc impl,
```

Comment:
This isn't true anymore, as this impl handles zero-sized layouts and then forwards to `GlobalAlloc`.